examples/sms/Receiving/main.py

```diff
@@ -21,14 +21,18 @@ class ReceiveSmsInterface(BoxLayout):
     pass
 
 
-class BroadcastButton(Button):
+class BroadcastButton(ToggleButton):
     def start(self):
-        SmsReceive.receive()
+        print "starting service"
```

Comment:
actually my code is not yet ready. I committed my current state of code just to show you so that you can understand the issues I am facing clearly. Cleaning of code is definitely required which I will do once my code is in merge able state. I used this print statement for debugging purpose.